virt/install/virt-planning-environment-object-maximums.adoc

```diff
@@ -0,0 +1,16 @@
+[id="virt-planning-object-maximums"]
```

Comment:
The ID and the context for the assembly should be the same value and mirror the file name as closely as possible.

Please change the ID to `[id="virt-planning-environment-object-maximums"]` and then also update the context to `:context: virt-planning-environment-object-maximums`

Reply:
ID and context have been updated to be consistent.